src/mcts/node.cc

```diff
@@ -74,13 +75,24 @@ class NodeGarbageCollector {
     while (!stop_.load()) {
       // Node will be released in destructor when mutex is not locked.
       std::unique_ptr<Node> node_to_gc;
+      size_t solid_size = 0;
       {
         // Lock the mutex and move last subtree from subtrees_to_gc_ into
         // node_to_gc.
         Mutex::Lock lock(gc_mutex_);
         if (subtrees_to_gc_.empty()) return;
         node_to_gc = std::move(subtrees_to_gc_.back());
         subtrees_to_gc_.pop_back();
+        solid_size = subtrees_is_solid_.back();
+        subtrees_is_solid_.pop_back();
+      }
+      // Solid is a hack...
+      if (solid_size != 0) {
+        for (int i = 0; i < solid_size; i++) {
```

Comment:
`for (size_t i = 0; i < solid_size; i++) {`

Reply:
Done.